packaging/dockerfiles/debian-buster.dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM debian:buster-slim
+LABEL maintainer "Hiroki Kumazaki <kumagi@google.com>"
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends -q \
```

Comment:
I'm not familiar with docker at all, but can you avoid writing these packages in every dockerfile? Obvious is to create a virtual debian package to install all of these and an apt repository that provides (only) that package, but I know it's way too much.